crates/sui-framework/packages/dwallet-system/sources/tendermint_lc.move

```diff
@@ -14,13 +13,28 @@ module dwallet_system::tendermint_lc {
     const TimestampInvalid: u64 = 5; 
     const NextValidatorsHashInvalid: u64 = 6; 
     const EUpdateFailed: u64 = 7;
-
+    
     // TODO: Make Client shareobject
     struct Client has key, store {
         id: UID,
-        latest_height: u64
+	chain_id: vector<u8>,
+	trust_threshold: u256,
```

Comment:
why do we need trust threshold? For tendermint it's always +2/3 by the voting power

Reply:
I checked tendermint-rs, and they also support 1/3 or arbitrary numbers.  Just ensure two popular cases.